dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="A library that implement a layer on top of PKCS#11 API"
```

Comment:
As DESCRIPTION I'd suggest to take it directly from the homepage. 1st paragraph of "PKCS#11" and then the "Overview":

> The PKCS#11 API is an abstract API to access operations on cryptographic objects such as private keys, without requiring access to the objects themselves. That is, it provides a logical separation of the keys from the operations.
> The PKCS #11 API is mainly used to access objects in smart cards and Hardware or Software Security Modules (HSMs). That is because in these modules the cryptographic keys are isolated in hardware or software and are not made available to the applications using them.
> 
> There are two libraries:
> 
> - libp11 provides a higher-level (compared to the PKCS#11 library) interface to access PKCS#11 objects. It is designed to integrate with applications that use OpenSSL.
> - pkcs11 engine plugin for the OpenSSL library allows accessing PKCS#11 modules in a semi-transparent way.
> 
> The wiki page for this project is at https://github.com/OpenSC/libp11/wiki and includes a bug tracker and source browser.